atomic_reactor/plugins/exit_delete_from_registry.py

```diff
@@ -42,64 +42,163 @@ def __init__(self, tasker, workflow, registries):
 
         self.registries = deepcopy(registries)
 
+    def setup_secret(self, registry, secret_path):
+        auth = None
+
+        if secret_path:
+            self.log.debug("registry %s secret %s", registry, secret_path)
+            dockercfg = Dockercfg(secret_path).get_credentials(registry)
+            try:
+                username = dockercfg['username']
+                password = dockercfg['password']
+            except KeyError:
+                self.log.error("credentials for registry %s not found in %s",
+                               registry, secret_path)
+            else:
+                self.log.debug("found user %s for registry %s", username, registry)
+                auth = requests.auth.HTTPBasicAuth(username, password)
+
+        return auth
+
+    def request_delete(self, url, manifest, insecure, auth):
+        response = requests.delete(url, verify=not insecure, auth=auth)
+
+        if response.status_code == requests.codes.ACCEPTED:
```

Comment:
I was talking about `response.ok`, not `response.status_code == requests.codes.ok`. Despite the naming, these are different: `response.ok` is only false when `raise_for_status()` would raise an exception.

That's what I meant about being strict. Currently we are checking for a particular response code, but I wonder if we should just accept any success code. We could do that with `if response.ok:`.